kernel/univ.ml

```diff
@@ -231,7 +231,7 @@ module LMap = struct
   module M = HMap.Make (Level)
   include M
 
-  let union l r = 
+  let lunion l r =
     merge (fun _k l r ->
```

Comment:
Could be defined using the new `union`

Reply:
I'm afraid not, look at the types.

Reply:
?
Isn't `lunion l r = union (fun _ a _ -> Some a) l r`?

Reply:
Somehow I thought I had tried that and failed. But it works, of course. Thanks! I also adapted `subst_union`.